### PR TITLE
Bella/gyms

### PR DESCRIPTION
**Pull request by isabellahoie**
Create Gym model, DayTime model, and the corresponding schema and resolvers for querying 

## Overview

 Created the gym and daytime models using sqlalchemy, set up flask app, set up sqlite database and session, created schema for the models and made resolvers for the queries.

## Changes Made

 Created sqlalchemy models based on the redesigned schema, set up the flask app, created a connection to the sqlite database and initialized the database; created schemas for each model and their resolvers for querying all the gyms and getting gyms by names, as well as getting gym times by day, start time, and end time 

## Test Coverage

 Tested features by creating gyms, gymtimes, and daytimes and storing them in the database. I then used graphiql to test queries: retrieving all gyms, gyms by name, and gym times by day, start times, and end times 

## Next Steps

I plan on creating the capacity model as well as activity model to continue implementing the Uplift schema using sqlalchemy models to store the data

### CODE REVIEW

File: src/app2/models/daytime.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from database import Base
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, Boolean, func
+from sqlalchemy.orm import backref, relationship
+from models.gym import Gym, GymTime
+
+class DayTime(Base):
+    __tablename__ = "daytime"
+
+    id = Column(Integer, primary_key=True)
+    day = Column(Integer, nullable=False)
```
Comment (markim21) [inline src/app2/models/daytime.py:11]: This is day of the week, right?

**Review by markim21 — APPROVED** [review]
lgtm!!! great work bella!!!

**Review by SashaLoayza — APPROVED** [review]

**Comment by SashaLoayza** [comment]
Are you aware that test/gyms fails?

**Comment by markim21** [comment]
> Are you aware that test/gyms fails?

yes, the tests are outdated. we haven't been able to create a working test suite with graphene (little documentation exists for it), and this task is low on the priority list for now. currently we test them locally by eye using graphiql to query our db.

**Pull request merged**